package.json

```diff
@@ -51,7 +51,9 @@
     "lint:changelog:rc": "auto-changelog validate --rc",
     "lint:eslint": "eslint . --ext js,ts,tsx,snap --cache",
     "lint:eslint:fix": "yarn lint:eslint --fix",
-    "lint:lockfile": "lockfile-lint --path yarn.lock --allowed-hosts npm yarn github.com codeload.github.com npm.pkg.github.com --empty-hostname false --allowed-schemes \"https:\" \"git+https:\"",
+    "lint:lockfile:dedupe": "yarn dedupe --check",
+    "lint:lockfile:dedupe:fix": "yarn dedupe --strategy highest",
```

Comment:
Nit: it looks like this strategy is the default (it's the only one implemented so far), so it's redundant to specify it. If we're going to include it, it should at least be set in all places `dedupe` is run (e.g. in the "check" script as well, and in CI config). But it seems easier to drop it here instead.

Reply:
Done in latest.